tests/integration/test_auditlogs.py

```diff
@@ -17,17 +16,24 @@
 
 
 @pytest.mark.integration
-@pytest.mark.parametrize(
-    "protocol", ["TCP", "UDP"],
-)
-def test_auditlogs_send_to_command_returns_success_return_code(
-    runner, integration_test_profile, protocol
+def test_auditlogs_send_to_tcp_command_returns_success_return_code(
+    runner, integration_test_profile, tcp_dataserver
```

Comment:
Do actually need to pass this in or will it work without it?

Reply:
I believe it should work without passing them in as fixture params

Reply:
The `tcp_dataserver` and `udp_dataserver` fixtures are what I am referring to

Reply:
If no tests use the fixture, it won't get set up unless we set `autouse=True` on it. Which we don't want, since we only want it to get set up on integration tests.